Assets/_BForBoss/_Utility/Scripts/Editor/PropertyDrawers/ResolverPropertyDrawer.cs

```diff
@@ -153,21 +153,24 @@ private ComponentInformation[] GetResolvedGameObjects(MonoBehaviour mb, ResolveT
             {
                 case ResolveType.FromChildren:
                 {
-                    foreach (Transform child in mb.transform)
+                    Transform[] transforms = mb.GetComponentsInChildren<Transform>(_includeInactiveGameObjects)
+                        .Where(transform => transform.gameObject != mb.gameObject).ToArray();
+
+                    foreach (Transform transform in transforms)
                     {
-                        components.Add(new ComponentInformation(child.gameObject));
+                        components.Add(new ComponentInformation(transform.gameObject));
                     }
                     
                     break;
                 }
                 case ResolveType.FromParent:
                 {
-                    Transform mbParent = mb.transform.parent;
+                    Transform[] transforms = mb.GetComponentsInParent<Transform>(_includeInactiveGameObjects)
```

Comment:
If I understand correctly, it finds the component within the Siblings of the parent?
A
  - B
   - - C [Resolve]
  - D
  - E
  - F

So it tries to find B, D, E F?

Reply:
Nope no siblings only the direct trail of parents (I.e. B and A, assuming B is a child of A)